addons/website/static/src/snippets/s_dynamic_snippet/options.js

```diff
@@ -4,9 +4,18 @@ odoo.define('website.s_dynamic_snippet_options', function (require) {
 const options = require('web_editor.snippets.options');
 
 const dynamicSnippetOptions = options.Class.extend({
+    /**
```

Comment:
> Prior to this commit, the Dynamic Snippet options would modify the DOM
at places were it's not supposed to. 

Is this a fix ? Stable ? Or is it an imp and it's ok without this in stable ? I think we talked about it but can't really remember, remind me :)

Reply:
Maybe mention that in the commit message (the PR that was for 15.0, the fact it was decided to do it master and why, etc :wink:)